dts/bindings/power-domain/power-domain-state-change-deviceonoff.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  This power domain will Turn On and Off devices when transitioning
+  in and out a specified Power State. Power States that trigger this
+  action is specified via the onoff-power-states property.
+
+compatible: "power-domain-state-change-deviceonoff"
```

Comment:
`deviceonoff` should be dropped from all naming, since that is just the definition of what a power domain does.

Reply:
Updated.